etherman/etherman.go

```diff
@@ -69,9 +71,15 @@ type ethClienter interface {
 	ethereum.LogFilterer
 	ethereum.TransactionReader
 	ethereum.ContractCaller
+	ethereum.GasPricer
 	bind.DeployBackend
 }
 
+type externalGasProviders struct {
+	EtherScan     etherscan.EtherscanI
+	EthGasStation ethgasstation.EthGasStationI
```

Comment:
Following the other component patterns, instead of having the interfaces for each package, like `etherscan.EtherscanI` and `ethgasstation.EthGasStationI`, we could have am `interfaces.go` file like this:

interfaces.go
```go
type gasPricer interface {
	// GasPrice returns the gas price
	GasPrice(ctx context.Context) (*big.Int, error)
}
```

and then on `etherman.go` we would have:

```go
type externalGasProviders struct {
	EtherScan     gasPricer
	EthGasStation gasPricer
}
```

usage:
```go
gas, err := etherMan.EtherScan.GasPrice()
```